src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Package/Update/PackageUpdateCommand.cs

```diff
@@ -34,9 +34,10 @@ internal static void Register(Command packageCommand, Func<ILoggerWithColor> get
         {
             var command = new DocumentedCommand("update", Strings.PackageUpdateCommand_Description, "https://aka.ms/dotnet/package/update");
 
-            var packagesArguments = new Argument<List<string>>("packages")
+            var packagesArguments = new Argument<List<Package>>("packages")
```

Comment:
@baronfel you'll notice I didn't add a `--version` option to this command, although `dotnet package add`, and probably others, support it.   What do you think?

The main reason I didn't is because `dotnet package update` will support upgrading multiple packages in one command, and while there will be situations where multiple packages will want to use the same version, it's not true in the general sense.  So, I was thinking to keep things simpler by not adding in the complication about what to do when multiple packages are provided, with one `--version`.

Reply:
Left some feedback on the spec.

I think this PR can be merged, and we can continue to sue the spec as a reference.